src/xmpp/uuid.go

```
@@ -15,3 +15,16 @@ func UUID4() string {
 	uuid[8] = (uuid[8] &^ 0x40) | 0x80
 	return fmt.Sprintf("%x-%x-%x-%x-%x", uuid[:4], uuid[4:6], uuid[6:8], uuid[8:10], uuid[10:])
 }
+
+func SessionId() string {
+	var strSize = 15
+	var dictionary string
+	dictionary = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
+
+	var bytes = make([]byte, strSize)
+	rand.Read(bytes)
```

This ignores errors.